src/node_messaging.cc

```diff
@@ -865,7 +865,12 @@ void MessagePort::Drain(const FunctionCallbackInfo<Value>& args) {
 }
 
 void MessagePort::ReceiveMessage(const FunctionCallbackInfo<Value>& args) {
-  CHECK(args[0]->IsObject());
+  Environment* env = Environment::GetCurrent(args);
+  if (!args[0]->IsObject() ||
+      !env->message_port_constructor_template()->HasInstance(args[0])) {
+    return THROW_ERR_INVALID_ARG_TYPE(env,
+        "First argument needs to be a MessagePort instance");
```

Comment:
To be consistent with the [doc](https://nodejs.org/dist/latest-v12.x/docs/api/worker_threads.html#worker_threads_worker_receivemessageonport_port), maybe change to:
```suggestion
        'The "port" argument must be a MessagePort instance');
```

Reply:
@ZYSzys Do you maybe want to open a PR after this lands that makes this consistent across both instances?

Reply:
Aha, I missed that there is still somewhere else need to be consistent.

Make sense to do it entirely after this lands.